cfy_manager/components/postgresql_client/postgresql_client.py

```diff
@@ -122,6 +126,13 @@ def configure(self):
     def remove(self):
         logger.notice('Removing PostgreSQL...')
         files.remove_notice(POSTGRESQL_CLIENT)
-        yum_remove('postgresql95')
-        yum_remove('postgresql95-libs')
+        rph = RpmPackageHandler(
+            'postgresql95-server-9.5.3-2PGDG.rhel7.x86_64.rpm')
```

Comment:
Mentioning rpm name create a very unique dependice outside, so I don't think this is a good idea

Reply:
This is the same in all other components' "remove" functions since we cannot rely on the config.yaml existing at this point